### ISSUE TITLE
Accessibility Bug Bash Info

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2915]
Hi everyone,

Welcome to the Accessibility Bug bash. Please note some of points applicable for this bug bash.

Applicable issues label: [Accessibility](https://github.com/topcoder-platform/community-app/issues?q=is%3Aissue+is%3Aopen+label%3AAccessibility)

Branch to use: [qa-accessibility](https://github.com/topcoder-platform/community-app/tree/qa-accessibility)

For more detail please check Challenge: https://www.topcoder.com/challenges/30095935

Note: Please don't use this feature in commit message. https://help.github.com/en/articles/closing-issues-using-keywords

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors Please join this bug bash to solve interesting issues and help our community app more accessible, primarily for members with impairment.

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors 

Please follow the bug bash rule. One of issue we are seeing is "Pull requests are not passing automated tests"
PR needs to pass all tests and lint issues before picking next ticket. 

Thanks for cooperation.

**Comment by mishacucicea** [comment]
@topcoder-platform/topcodercompetitors
For the issues labeled with Login/Registration the repo to use is  https://github.com/appirio-tech/accounts-app and the `qa-accessibility` branch.
Existing PRs to the `dev` branch need to be recreated. I'm apologize for the inconvenience.

**Comment by sumitdaga** [comment]
@mishacucicea  
is it ok to just change the base branch ? 
it seems ok in my case https://github.com/appirio-tech/accounts-app/pull/231  
let me know if i should create a new PR?

**Comment by mishacucicea** [comment]
@sumitdaga It's perfect. The `dev` and `qa-accessibility` are basically the same, but we want to keep track of these items separately.

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors 

For contrast related issues, please get help from @Dara-K for selecting the correct color(if color is branding colors of topcoder, e.g. member rating, theme colors)

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors

Please note that we are targeting minimum **WCAG level AA**

**Comment by codeMinter** [comment]
Steps to run the app locally (will help if this is your first time working on this repo):
```
1. nvm use
2. npm i
3. npm run build:dev
4. npm run dev
5. go to localhost:3000 in browser - you will be redirected to login page, login with username / pwd shared in forum, browser will redirect to topcoder-dev post login.
6. Change url again to localhost:3000, this time it will not not ask to login and show quick links page
```

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors

We still have more than `$900` in purse, and we have merged few code, so chances of remaining issue is already fixed is high, pick the issue and if you find that this already fixed by another issue you got $$ just referencing any pull requests that fixes(whether you fixed that or other had).

Let's  crush the remaining issues and help member with less capability to access our app more easily.

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors

The prize amount is doubled for these remaining issues.

Issue prize ranges from minimum $20 to $80, please join to get higher prize.  Thanks!

**Comment by veshu** [comment]
Test Accounts for dev : pshah_manager/topcoder123 | pshah_copilot/topcoder123 | pshah_customer/topcoder123 | pat_monahan/topcoder123!
Ghostar/appirio123

**Comment by codeMinter** [comment]
@topcoder-platform/topcodercompetitors please keep an eye on your branch conflicts and resolve them ASAP for PRs to be reviewed and merged. Thanks

**Comment by gets0ul** [comment]
> The prize amount is doubled for these remaining issues.

Why are for some issues, they stay the same, not doubled?

**Comment by veshu** [comment]
@gets0ul Please help me point out the specific issues. May be some issue which were open later were not updated.

@PrakashDurlabhji You have lots of feedback issue to fix. Please submit the update on PR as soon as possible, if you are not available please remove assignment and add open for pickup. Thanks!

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors please make sure new PR udpate is using
"topcoder-react-ui-kit": "^1.0.2",

Make sure you updated the community-app repo. Thanks!

**Comment by PrakashDurlabhji** [comment]
@veshu yes I am updating all, and any issues made open which were assigned to me? I see 3 issues are made open but dont know which one.

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors

Please check here if your PR is failed, check earliest and remove the label if you update the PR.

https://github.com/topcoder-platform/community-app/labels/Code%20Review%20Failed

Otherwise we will open the ticket for open.

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors 
Hello members, we have last 10 issues for open pickup, let us know if anything. Suggest your prize to fix this we can accept if justifiable amount and effort. Thanks!

https://github.com/topcoder-platform/community-app/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3AAccessibility+label%3Atcx_OpenForPickup

**Comment by veshu** [comment]
@topcoder-platform/topcodercompetitors
Hello members,  We have new set of accessibility issues with good prize and easy fixes https://github.com/topcoder-platform/community-app/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Atcx_OpenForPickup+label%3AAccessibility with total prize to grab $275+ 

Join the bug bash and help Topcoder more accessible to differently able.
